service/src/main/java/bio/terra/workspace/service/resource/ResourceValidationUtils.java

```diff
@@ -377,6 +387,16 @@ public static void validateStorageAccountName(String storageAccountName) {
     }
   }
 
+  public static void validateStorageContainerName(String storageContainerName) {
+    if (!AZURE_STORAGE_CONTAINER_NAME_VALIDATION_PATTERN.matcher(storageContainerName).matches() ||
+            storageContainerName.contains("--")) {
```

Comment:
My regexp skills aren't strong enough to figure out how to enforce no consecutive dashes in the regexp itself.